array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
+function take(array, count) {
+  if (array.length > 0) {
+    array.length = count;
```

Comment:
This mutates the original array, and should return a new array.  You should use a loop in here.